src/components/forms/PrescriptionForm.tsx

```diff
@@ -144,11 +144,23 @@ const PrescriptionForm: FunctionComponent<PrescriptionFormProps> = ({
     }
   }
 
+  const updatePatientDate = (date: Date, field: string): void => {
+    const formattedDate = format(date, 'yyyy-MM-dd')
```

Comment:
DRY date format also used in src/components/forms/PatientForm.tsx

Reply:
done

Reply:
Use `STANDARD_DATE_FORMAT` here